homeassistant/components/calendar/todoist.py

```diff
@@ -217,10 +224,34 @@ def handle_new_task(call):
             date_format = '%Y-%m-%dT%H:%M'
             due_date = datetime.strftime(due_date, date_format)
             item.update(due_date_utc=due_date)
-        # Commit changes
-        api.commit()
+
+        # Commit task
+        result = api.commit()
         _LOGGER.debug("Created Todoist task: %s", call.data[CONTENT])
 
+        # Add default reminder to the task if user has default reminders set
+        user = api.user.get()
```

Comment:
This should be only fetched once and cached if user is premium

Reply:
Done